package.json

```diff
@@ -5,7 +5,8 @@
   "main": "dist/banana-i18n.js",
   "scripts": {
     "test": "NODE_ENV=test mocha",
-    "build": "webpack --mode=production"
+    "build": "webpack --mode=production",
+    "prepare": "npm run build"
```

Comment:
This addition was only needed to allow me to install my forked version of this library directly from GitHub.